src/high_level.rs

```diff
@@ -616,24 +617,46 @@ pub fn look_for_dep_with_data_hash(data_hash: &[u8]) -> Result<usize, SysError>
     look_for_dep_with_hash2(data_hash, ScriptHashType::Data)
 }
 
+pub fn encode_hex(data: &[u8]) -> CString {
+    let mut s = String::with_capacity(data.len() * 2);
```

Comment:
```suggestion
    let mut s = String::with_capacity(data.len() * 2 + 2);
```

We have 0x prefix, right?

Reply:
`encode_hex` will not add 0x